tests/conformance/conformance_test.go

```diff
@@ -788,7 +789,7 @@ func fsHeaderForEntry(hdr *tar.Header) FSHeader {
 		Name:     hdr.Name,
 		Linkname: hdr.Linkname,
 		Size:     hdr.Size,
-		Mode:     hdr.Mode,
+		Mode:     (hdr.Mode & int64(fs.ModePerm)),
```

Comment:
```suggestion
		Mode:     (hdr.Mode & int64(os.ModePerm)),
```
In newer releases, `os.ModePerm` is an alias for `fs.ModePerm`, but this branch used Go 1.13.

Reply:
I never would have guessed that.  Thanks!